src/test/java/checker/MiLoGCheckerDayTimeExceedanceTest.java

```diff
@@ -0,0 +1,261 @@
+package checker;
+
+import static org.junit.Assert.*;
+
+import java.time.LocalDate;
+import java.time.Month;
+import java.time.YearMonth;
+import java.util.Random;
+
+import org.junit.Test;
+
+import data.Employee;
+import data.Entry;
+import data.Profession;
+import data.TimeSheet;
+import data.TimeSpan;
+import data.WorkingArea;
+
+public class MiLoGCheckerDayTimeExceedanceTest {
```

Comment:
Is the check successful when there are zero entries on the timesheet?

Reply:
Of course it should be, but I will add this test in order to prove this behaviour.